crates/humanode-runtime/src/lib.rs

```diff
@@ -385,9 +385,23 @@ impl pallet_grandpa::Config for Runtime {
 /// A timestamp: milliseconds since the unix epoch.
 pub type UnixMilliseconds = u64;
 
+parameter_types! {
+    pub storage EnableDevBlockImportSeal: bool = false;
+}
+
+pub struct ConsensusOnTimestampSet<T>(PhantomData<T>);
+impl<T: pallet_babe::Config> OnTimestampSet<T::Moment> for ConsensusOnTimestampSet<T> {
+    fn on_timestamp_set(moment: T::Moment) {
+        if EnableDevBlockImportSeal::get() {
+            return;
+        }
+        <pallet_babe::Pallet<T> as OnTimestampSet<T::Moment>>::on_timestamp_set(moment)
+    }
+}
```

Comment:
Interesting, why is this needed?

Reply:
Do we maybe want to have a second, different runtime for the instant/manual seal consensus, without babe and grandpa pallets/APIs?